src/api/goods.ts

```diff
@@ -1,11 +1,17 @@
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
-export function getAll(): Promise<Good[]> {
-  return fetch(API_URL)
-    .then(response => response.json());
-}
+export const getAll = async (): Promise<Good[]> => {
+  return (await fetch(API_URL)).json();
```

Comment:
такая запись места занимает места, но читаемость и поддерживаемость на низком уровне

best way
```suggestion
  const response = await fetch(API_URL);
  
  return response.json();
```